<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresetWidget</class>
 <widget class="QWidget" name="PresetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>32</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>32</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>77</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>MonitorLabel</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="presetButton">
     <property name="toolTip">
      <string>activate preset for</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="preset">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>114</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>preset value for </string>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>2000000000</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="resetButton">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>24</height>
      </size>
     </property>
     <property name="toolTip">
      <string>reset </string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="currentValue">
     <property name="minimumSize">
      <size>
       <width>84</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>123456789012</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="rate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>56785678</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>presetButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>PresetWidget</receiver>
   <slot>presetCheckClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>PresetWidget</receiver>
   <slot>resetButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>resetButtonClicked()</signal>
  <signal>presetCheckClicked(bool)</signal>
 </slots>
</ui>
